modules/addBook.js

```diff
@@ -0,0 +1,24 @@
+export default (BookList) => {
+  const addbtn = document.getElementById('add-btn');
+  const form = document.querySelector('.form');
+  const err = document.getElementById('error-msg');
+
```

Comment:
- [ ] Dear @ankitt26, Please take a look into the `BookList` functionality of the app. Currently, I can use the add book without error, but the added book is not displayed in the `List` menu. we need to make sure the app is working - it is possible to add and remove books from the list.

### Video description of the problem.
[Screencast from 01-05-2023  4:44:13 ከሰዓት.webm](https://user-images.githubusercontent.com/69077061/235460322-86d7ecb3-4f21-43ed-b110-dcba7997e79f.webm)

Reply:
Thanks for video